src/charm.py

```diff
@@ -170,6 +191,10 @@ def _command(self) -> str:
 
             args.append(f"--web.external-url={external_url}")
 
+        # enable remote write if an instance of the relation exists
+        if self.model.relations["remote-write-server"]:
```

Comment:
iirc, this can raise KeyError when no relation present.
```suggestion
        if "remote-write-server" in self.model.relations:
```

Reply:
I'm pretty sure it doesn't.